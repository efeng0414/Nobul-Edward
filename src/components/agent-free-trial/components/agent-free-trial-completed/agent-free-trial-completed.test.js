import React from "react";
import renderer from "react-test-renderer";
import { IntlProvider } from "react-intl";
import AgentFreeTrialCompletedWrapped from "./index";

describe("AgentFreeTrialAccountInfo Component", () => {
  const {
    WrappedComponent: AgentFreeTrialCompleted
  } = AgentFreeTrialCompletedWrapped;
  const componentProps = {
    intl: global.getIntl(),
    firstName: "Agent"
  };

  const componentJSX = <AgentFreeTrialCompleted {...componentProps} />;

  describe("Component Snapshot", () => {
    const snapshotJSX = (
      <IntlProvider {...global.intl}>{componentJSX}</IntlProvider>
    );
    const componentSnapshot = renderer.create(snapshotJSX).toJSON();
    it("Should match stored snapshot", () => {
      expect(componentSnapshot).toMatchSnapshot();
    });
  });
});
