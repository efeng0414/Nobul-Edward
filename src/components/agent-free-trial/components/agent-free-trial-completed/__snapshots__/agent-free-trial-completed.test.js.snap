// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AgentFreeTrialAccountInfo Component Component Snapshot Should match stored snapshot 1`] = `
<div>
  <div
    className="free-trial-info-title"
  >
    Hello Agent,
  </div>
  <div
    className="free-trial-info-completed-title"
  >
    You are no longer a Nobul Premium member
  </div>
</div>
`;
